app/src/org/commcare/android/models/AsyncEntity.java

```diff
@@ -115,6 +115,9 @@ public Object getField(int i) {
                 } catch(XPathException xpe) {
                     xpe.printStackTrace();
                     data[i] = "<invalid xpath: " + xpe.getMessage() + ">";
+                } catch (XPathSyntaxException xse) {

```

Comment:
Good point, it'd be better to catch these errors earlier. Will do a separate PR.